services/API-service/src/api/event/event-place-code.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, Check, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('event_place_code', { schema: 'IBF-pipeline-output' })
+export class EventPlaceCodeEntity {
+  @PrimaryGeneratedColumn()
+  public id: number;
+
+  @Column({})
+  public placeCode: string;
+
+  @Column({ type: 'timestamp' })
+  public startDate: Date;
+
+  @Column({ type: 'timestamp', nullable: true })
+  @Check(`"startDate" < "endDate"`)
+  public endDate: Date;
+
+  @Column({ type: 'timestamp', nullable: true })
+  public manualClosedDate: Date;
+
+  @Column({ default: true })
```

Comment:
not sure if this should be `true` by default

Reply:
It should be because events are create, because there is an active trigger.